frontend-react/src/pages/submissions/SubmissionTable.test.tsx

```diff
@@ -73,4 +74,28 @@ describe("SubmissionTable", () => {
         const rows = within(tBody).getAllByRole("row");
         expect(rows).toHaveLength(2);
     });
+
+    describe("when rendering as an admin", () => {
+        beforeEach(() => {
+            mockSessionContext.mockReturnValue({
+                activeMembership: {
+                    memberType: MemberType.PRIME_ADMIN,
+                    parsedName: Organizations.PRIMEADMINS,
+                    service: "",
+                },
+                dispatch: () => {},
+                initialized: true,
+            });
+
+            renderWithResolver(<SubmissionTable />, []);
+        });
+
+        test("renders a warning about not being able to request submission history", async () => {
```

Comment:
This test outputs a test warning about state changes because of the [`dispatch` in usePagination](https://github.com/CDCgov/prime-reportstream/blob/master/frontend-react/src/hooks/UsePagination.ts#L397).  I don't think this can be avoided without making further changes to our source code so my feeling is that it's a tolerable warning.  I also feel like the usePagination hook could be replaced with a custom useQuery so we're not having to maintain the event bus -- but this is also outside the scope of this pull request.  I'll add a ticket or two with some thoughts.